output/schema/schema.json

```diff
@@ -110992,6 +110992,24 @@
       },
       "specLocation": "eql/search/types.ts#L20-L32"
     },
+    {
+      "kind": "enum",
+      "members": [
+        {
+          "description": "Run against the first version of ESQL.",
+          "name": "2024.04.01"
+        },
+        {
+          "description": "Run against the unreleased version of the ESQL language. This will\ncontain any changes staged for release that have yet to be included\nin a version.",
```

Comment:
IMHO we should mention that this only works on SNAPSHOT builds. ES on non-SNAPSHOT builds will return a 400 if it sees `"version": "snapshot"`.

I wonder if we should even make `snapshot` a part of the official specs - it's not like users miss out if they cannot use language feature xyz, given that existing language versions are going to still be updated with new features.

Can we maybe add `snapshot` here later, if it turns out that there's an important enough use case for it?

Reply:
Yeah, I'll drop it.